client/packages/system/src/Patient/CreatePatientModal/CreatePatientModal.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { FC, useEffect, useState } from 'react';
```

Comment:
moved a few things into here - mainly the modal control so that I could use the buttons to update the tabs.